ElfUtils/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(ElfUtils)
+add_library(ElfUtils STATIC)
+
+target_compile_options(ElfUtils PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_sources(
+  ElfUtils
+  PUBLIC include/ElfUtils/ElfFile.h)
+
+target_sources(
+  ElfUtils
+  PRIVATE ElfFile.cpp)
+
+target_include_directories(ElfUtils PUBLIC ${CMAKE_CURRENT_LIST_DIR}/include)
+
+target_link_libraries(
+  ElfUtils
+  PUBLIC OrbitBase
+         OrbitProtos
+         abseil::abseil
+         llvm_object::llvm_object
+         Outcome::Outcome)
+
+add_executable(ElfUtilsTests)
+
+target_sources(ElfUtilsTests PRIVATE
+    ElfFileTest.cpp
+)
+
+target_link_libraries(
+  ElfUtilsTests
+  PRIVATE ElfUtils
+          OrbitCore # TODO: This is needed for GetExecutablePath consider moving Path.h to OrbitBase
```

Comment:
`Path` would really need to get a serious cleanup before moving it to `OrbitBase` though.

Reply:
agree I was thinking about moving one function at the time adding tests and rewriting it if necessary.